### ISSUE TITLE
settings.txt does not exist

### ISSUE THREAD

**Issue by royfrancis** [yarden/MISO#104]
Hi,
I get the following error when I run MISO.

`Error: Settings file /home/rmf/.pyenv/versions/2.7.6/lib/python2.7/site-packages/misopy/settings/miso_settings.txt does not exist.`

I am running my code like so:

```
miso \
--run="$path/miso-index/" \
--read-len=126 \
--use-cluster \
--paired-end=PAIRED_END \
--output-dir="something-miso" \
--prefilter \
something.bam
```
I think my python environment is ok.

```
matplotlib==2.0.2
misopy==0.5.4
numpy==1.13.1
pandas==0.19.2
```

`samtools` and `bedtools` are loaded as well.

I was wondering if you might have any ideas on what might be wrong.

**Comment by sepidehparhami** [comment]
I ran into this problem too; a quick fix is to copy over the miso_settings.txt from this repo into ./misopy/settings on your computer

**Comment by yarden** [comment]
This is an indication that the package was not installed correctly; settings.txt should be available as data from the installed package directory. Did you install with a package manager like ``pip``?

**Comment by royfrancis** [comment]
Yes. It was installed using `pip`. I am using python using pyenv on a cluster. It was complaining this does not exist `/home/rmf/.pyenv/versions/2.7.6/lib/python2.7/site-packages/misopy/settings/miso_settings.txt`. So I checked at this location and the `miso_settings.txt` and the settings directory was missing. I created the directory and copied it from this website as suggested by @sepidehparhami. 

Now, it complains about output folder although it has been provided (as shown in my first comment).
`Error: need --output-dir to compute Psi values.`

**Comment by chonga2022** [comment]
did you solve this error？

**Comment by chonga2022** [comment]
I got the same error
